cmd/image-inspector.go

```diff
@@ -16,7 +16,7 @@ func main() {
 	flag.StringVar(&inspectorOptions.DstPath, "path", inspectorOptions.DstPath, "Destination path for the image files")
 	flag.StringVar(&inspectorOptions.Serve, "serve", inspectorOptions.Serve, "Host and port where to serve the image with webdav")
 	flag.BoolVar(&inspectorOptions.Chroot, "chroot", inspectorOptions.Chroot, "Change root when serving the image with webdav")
-	flag.StringVar(&inspectorOptions.DockerCfg, "dockercfg", inspectorOptions.DockerCfg, "Location of the docker configuration file")
+	flag.StringVar(&inspectorOptions.DockerCfg, "dockercfg", inspectorOptions.DockerCfg, "Location of the docker configuration files seperated by ',' if more than one")
```

Comment:
spf13/pflags makes it easy with `StringSliceVar` https://github.com/spf13/pflag/blob/master/string_slice.go#L67.  If we don't want to pull in that dependency it would have to be a custom `Var` with a set implementation.